### DISCUSSION TITLE
Face similarity - distance to percent of similarity?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18195] [Q&A and General discussion] [answered]
Face similarity - distance to percent of similarity?
Is there a good formula to convert distance to percent of similarity for image net?

["['(distance: 0.4115199148654938, id: 4)', '(distance: 0.4121388792991638, id: 41)', '(distance: 0.5040524005889893, id: 14)', '(distance: 0.5864538550376892, id: 7)', '(distance: 0.8445382118225098, id: 19)']"]

First 3 result are for the same person, another one to a different one.

**Comment by yhmo** [comment, marked as answer]
I think it is hard to find a way to convert distance to percent of similarity. We don't have a proper denominator to calculate the percent value.
